pkgs/development/python-modules/tensorboard/default.nix

```diff
@@ -26,7 +26,7 @@ buildPythonPackage rec {
   pname = "tensorboard";
   version = "2.11.0";
   format = "wheel";
-  disabled = pythonOlder "3.6" || pythonAtLeast "3.11";
```

Comment:
Why is it disabled for python >=3.11? When can we remove this constraint? Let's please add a comment here clarifying what's going on (ideally with a link to a GitHub issue, etc)

Reply:
I don't know. From what I see, what breaks it is having it disabled because it builds for Python 3.11. I'm not aware of a GitHub issue related. I have proposed what works for me. Having this package disabled breaks other python packages for python 3.11, such as `torchaudio`.

Reply:
I have taken into consideration your comment of: "add a comment here clarifying what's going on"
But I don't know what to say. I have nothing to say. I suppose nothing is better than noise? I'd rather default to not say anything once I don't have anything to say. So for now, I consider the matter of lack of comment settled [on my part at least].
Let me know the content of clarification if necessary.